src/deploy/osp_deployer/profiles/csp.json

```diff
@@ -22,7 +22,10 @@
             "sah_bond_opts": "",
             "nic_env_file": {
                "valid_values": [
-                  "5_port/nic_environment.yaml"
+                  "5_port/nic_environment.yaml",
+                  "dvr_5_port/nic_environment.yaml",
+                  "dvr_7_port/nic_environment.yaml"
```

Comment:
Is there any reason to not allow the 4 port and 6 port configuration in the CSP profile?

Reply:
We didn't add the 4 & 6 port configs, because the original file contained only 5_port.

Reply:
Ok.  I don't see any reason not to add them, so can you add them here please?

Reply:
Will do.